### DISCUSSION TITLE
long youtube load on beta.227

### DISCUSSION THREAD

**Discussion by tadmi** [vuepress-theme-hope/vuepress-theme-hope#3243] [Q&A] [answered]
[On the version 2.0.0-beta.227, there may be a problem with a long YouTube video load](https://theme-hope.vuejs.press/guide/markdown/components.html#youtube)

**Comment by Mister-Hope** [comment, marked as answer]
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/3d648d6d-7398-4a01-bf74-1b63cc5bee3e)

It's ok on my device, we only ensure that iframe is loaded before showing that to you.

The page may be problematic, as there are too many heavy iframes on that page. So you may need to wait a long time for the upper iframes to be loaded before loading the bottom youtube iframe
